miner/src/miner.rs

```diff
@@ -70,11 +71,34 @@ impl Miner {
                 .timestamp(current_time)
                 .parent_hash(parent_hash);
 
+            let uncles = uncles.into_iter().map(TryInto::try_into).collect();
+            if let Err(ref e) = uncles {
+                error!(target: "miner", "error parsing uncles: {:?}", e);
+            }
+            let cellbase = cellbase.try_into();
+            if let Err(ref e) = cellbase {
+                error!(target: "miner", "error parsing cellbase: {:?}", e);
```

Comment:
Should return a Result instead continue execution?
The below code will panic

Reply:
Ouch I meant to return after printing log here, somehow forgot to add the code 😂 Thanks for pointing this out

Reply:
Consider using the `?` operator to return Result? which is a more general way to do error handling in rust, current code looks like golang style..

Reply:
How about now?